addons/mail/static/src/models/emoji_search_bar_view.js

```diff
@@ -79,6 +79,9 @@ registerModel({
         }),
         placeholder: attr({
             compute() {
+                if (this.emojiPickerView.hoveredEmojiPickerView) {
+                    return this.emojiPickerView.hoveredEmojiPickerView.emoji.shortcodes
```

Comment:
`shortcodes` is an array of strings, while `placeholder` is a string.
Should `shortcodes.map()` and make a proper string out of it, rather than rely on `Array.toString()`.

Also the visual should work in different languages, and separator is locale dependent.
Should join with `this.env._t(", ")`, as separator might be different in some languages (notably `、` in Japanese)

Reply:
And example of emoji with several shortcodes:
<img width="301" alt="Screenshot 2022-11-01 at 21 40 01" src="https://user-images.githubusercontent.com/6569390/199336559-e43eecad-c99c-41c4-a067-255b698dd55b.png">

Speaking of which... A bit strange that we don't have `:smiling_face_with_halo:` listed here, which is apparently an expected shortcode for this emoji:
https://emojipedia.org/smiling-face-with-halo/

Reply:
> Speaking of which... A bit strange that we don't have `:smiling_face_with_halo:` listed here, which is apparently an expected shortcode for this emoji: https://emojipedia.org/smiling-face-with-halo/

There is nothing to be expected as for the shortcodes, since there is no such thing as a standard for them. The current codes were generated from their English CLDR name, and for a handful of them I tailored the shortcode by hand to have something a bit less contrived, which is most likely what happened to `smiling face with halo`.

Reply:
> Also the visual should work in different languages, and separator is locale dependent. Should join with `this.env._t(", ")`, as separator might be different in some languages (notably `、` in Japanese)

Shortcodes will be in English anyway, so it wouldn't make much sense to me to use a Japanese comma as a separator. I'd rather go for a simple space, as what is done in Discord. What do you think, Alex?
![image](https://user-images.githubusercontent.com/20225337/199449210-fe13db9f-e37b-4ce9-b17a-97d79a0e6f5a.png)

Reply:
Indeed makes sense to just whitespace and no local, since as you said it's same value in english anyway.